src/stylesheets/components/_hero.scss

```diff
@@ -2,36 +2,45 @@
 // ==========================
 
 .usa-hero {
-  @include u-padding-y($site-margins);
+  @include typeset;
+  @include u-padding-y(units($theme-site-margins));
   background-image: url('#{$theme-image-path}/hero.png');
   background-position: center;
   background-size: cover;
 
   .usa-button {
-    color: $color-white;
+    color: color('white');
+  }
+
+  p {
+    @include typeset-p;
   }
 }
 
 .usa-section-dark {
   .usa-hero-link {
-    color: $color-primary-alt;
+    color: color('accent-cool');
   }
 }
 
 .usa-hero-callout {
-  background-color: $color-primary-darkest;
-  padding: 3rem;
+  background-color: color('primary-darker');
+  padding: units(4);
 
-  @include media($small-screen) {
+  @include at-media('mobile-lg') {
     max-width: 30rem;
   }
 
   > *:first-child {
-    @include margin(0 null $spacing-medium null);
+    margin-top: 0;
   }
 }
 
 .usa-hero-callout-alt {
-  color: $color-white;
+  color: color('white');
```

Comment:
What's `usa-hero-callout-alt` used for? I don't see it in the example

Reply:
The heading has two colors: white and blue:

<img width="267" alt="screen shot 2018-09-12 at 11 38 31 am" src="https://user-images.githubusercontent.com/5249443/45445903-6b799780-b680-11e8-86e6-6f9588e83789.png">

Reply:
It's in the code https://github.com/uswds/uswds/pull/2726/files#diff-d55517edab231c0570e713ee03a4d9dbR7

Reply:
oh yeah duh 🙃